test/runtests.jl

```diff
@@ -1,3 +1,4 @@
+using Arrow
```

Comment:
```suggestion
VERSION >= v"1.3" && using Arrow
```

Reply:
Would allow the tests to work if you work on Julia 1.0 if you work around Arrow.jl requiring 1.3

Reply:
It still wouldn't resolve to hit this anyways cause `Pkg.test` would error trying to resolve the extra dependency. If `Pkg` allowed you to conditionally install packages then this would work.

Reply:
I know. This just helps for when someone manually removes `Arrow` from the test dependency list and runs the tests.

Reply:
👌🏻